templates/base.html

```diff
@@ -68,19 +68,19 @@
     {% endblock %}
 
 
-    {# Twitter cards #}
-    <meta property="twitter:domain" content="{{app.site.url}}/">
-    {% if request.GET.twittercard == 'large' %}
-        <meta property="twitter:card" content="summary_large_image">
+    {# X cards #}
+    <meta property="X:domain" content="{{app.site.url}}/">
+    {% if request.GET.Xcard == 'large' %}
+        <meta property="X:card" content="summary_large_image">
     {% else %}
-        <meta property="twitter:card" content="summary">
+        <meta property="X:card" content="summary">
```

Comment:
```suggestion
        <meta property="twitter:card" content="summary">
```

Le nom des paramètres `meta` [n’a pas changé](https://developer.x.com/en/docs/twitter-for-websites/cards/overview/markup).